README.md

```diff
@@ -0,0 +1,47 @@
+# ScrapingBee Python SDK
+
+[ScrapingBee](https://www.scrapingbee.com/) is a web scraping API that handles headless browsers and rotates proxies for you. The Python SDK makes it easier to interact with ScrapingBee's API.
+
+## Installation
+
+You can install ScrapingBee Python SDK with pip.
+
+```bash
+pip install scrapingbee
+```
+
+## Usage
+
+The ScrapingBee Python SDK is a wrapper around the [requests](https://docs.python-requests.org/en/master/) library. ScrapingBee supports GET and POST requests.
+
+Signup to ScrapingBee to [get your API key](https://app.scrapingbee.com/account/register) and some free credits to get started.
+
+### Making a GET request
+
+```python
+>>> from scrapingbee import ScrapingBeeClient
+
+>>> scrapingbee = ScrapingBeeClient(api_key='REPLACE-WITH-YOUR-API-KEY')
```

Comment:
Using `scrapingbee` as a variable overrides the package name. I think it's better to change it to:

```
client = ScrapingBeeClient(api_key='REPLACE-WITH-YOUR-API-KEY')
```